ui/src/components/ReleaseCard/index.jsx

```diff
@@ -254,7 +254,9 @@ function ReleaseCard(props) {
                               clickable
                               size="small"
                               icon={<LinkIcon />}
-                              label={ruleId}
+                              label={`${ruleInfo.product} ${
+                                ruleInfo.channel ? `- ${ruleInfo.channel}` : ''
```

Comment:
This is not what we want either. The alias is a property of the `rule`. I can see why you're struggling though - the alias is not returned by the `/releases` endpoint, so the data you need is not even present.

I suggest you fetch the rules (similar to how we do on the rules page: https://github.com/mozilla-releng/balrog/blob/ca55f9aa6aa15647370a841d8827e1e724ab8813/ui/src/views/Rules/ListRules/index.jsx#L164), and use that to find the alias for each ruleId.